### PR TITLE
ITOS-9: added ccd import service

### PR DESCRIPTION
**Pull request by pawelcieszko**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.io.IOException;
+
+public class XdsCcdImporterImpl implements XdsCcdImporter {
+
+    @Autowired
+    @Qualifier("ccdService")
+    CcdService ccdService;
```
Comment (dserkowski) [inline api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java]: Add 'private' here

**Review by dserkowski — CHANGES_REQUESTED** [review]

**Comment by pawelcieszko** [comment]
Due to sedish and Haiti server malfunctions I wasn't able to test it properly so I suggest merging it after some testing.

File: api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import org.openmrs.module.xdssender.api.service.CcdService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.io.IOException;
+
+public class XdsCcdImporterImpl implements XdsCcdImporter {
+
+    @Autowired
+    @Qualifier("ccdService")
```
Comment (dserkowski) [inline api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java]: I think that you should use the convention "artifactId.className"

**Review by dserkowski — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java (hunk 2 above)
Comment (pawelcieszko) [reply api/src/main/java/org/openmrs/module/registrationcore/api/xdssender/impl/XdsCcdImporterImpl.java]: Right, this qualifier has been already defined. Nevertheless, I will correct it.

**Comment by pawelcieszko** [comment]
Requires xdssender ccd changes before merging

**Pull request merged**